### PR TITLE
Update Repository Readme

### LINKED ISSUE #8: Setup repository
## Description

Setup the correct, creating a README with useful content for anyone to read about the project.

## Acceptance Criteria

- [ ] README with the description of the project, team outline, contributing (must include design and codebase information), milestones, and any links to documents such as the papers and the IDM Workplan & IDM Breakdown documents
- [ ] Setup the design folder structure based on the design naming guidelines
- [ ] Describe the work process which summarises the agile methodology we are using, the sprint moments and how the user-stories/issues are organised

### PR DESCRIPTION
**Pull request by paulobmarcos**
This PR updates the repository readme to a more complete description of the project.

Closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
+The Identity Manager - IDM for short - is a unified digital wallet based on open-standards that aims to support multiple decentralized identities.
 
-[![#ipfs-identity](https://img.shields.io/badge/irc-%23ipfs--identity-brightgreen.svg)](https://webchat.freenode.net/?channels=ipfs-identity)
+## Documents
 
-## Current status
+Here is a list of some useful documents. If you're interested in collaborating, please check [Contributing](#contributing).
 
-[![Waffle.io - Columns and their card count](https://badge.waffle.io/ipfs-shipyard/pm-idm.svg?columns=all)](https://waffle.io/ipfs-shipyard/pm-idm)
+[💡 Concept](docs/idm-concept.md)
```
Comment (satazor) [inline README.md]: This should be a list or add 2 empty spaces in the end to make it a new line.
Also, you may add the "Specification (wip)" pointing to https://cryptpad.fr/code/#/2/code/view/x6emT-0hXpN4eg-9wZEHaKsmgOEU+jdcyBx9-pG6nQQ/

File: README.md [hunk 2]
```diff
@@ -1,9 +1,54 @@
...
+- [André Cruz](https://github.com/satazor) - Software engineer
+- [André Sousa](https://github.com/andreforsousa) - Designer
+- [Marco Oliveira](https://github.com/marcooliveira) - Product engineer
+- [Paulo Marcos](https://github.com/paulobmarcos) - Software engineer
+- [Pedro Santos](https://github.com/PedroMiguelSS) - Software engineer
+- [Pedro Teixeira](https://github.com/pgte) - Adviser
+
+## Milestones & Progress
+
+Our milestones are continually updated and detailed in the [IDM Workplan Document](https://docs.google.com/spreadsheets/d/1Venqgkcao2Lcje0mkCxr9u0H037aC3H5IxMfsoaeMoE).
```
Comment (satazor) [inline README.md]: continually -> continuously?

File: README.md [hunk 3]
```diff
@@ -1,9 +1,54 @@
...
-[![#ipfs-identity](https://img.shields.io/badge/irc-%23ipfs--identity-brightgreen.svg)](https://webchat.freenode.net/?channels=ipfs-identity)
+## Documents
 
-## Current status
+Here is a list of some useful documents. If you're interested in collaborating, please check [Contributing](#contributing).
 
-[![Waffle.io - Columns and their card count](https://badge.waffle.io/ipfs-shipyard/pm-idm.svg?columns=all)](https://waffle.io/ipfs-shipyard/pm-idm)
+[💡 Concept](docs/idm-concept.md)
+[🛠 IDM Breakdown](https://docs.google.com/document/d/1g0TjSPjEM4pryPwJTGhIeE4DBsj-VJpz_JqbfllJUgA)
+[🗓 IDM Workplan](https://docs.google.com/spreadsheets/d/1Venqgkcao2Lcje0mkCxr9u0H037aC3H5IxMfsoaeMoE)
```
Comment (paulobmarcos) [inline README.md]: The documents all being displayed in a single line. Should be separate lines.

File: README.md (hunk 2 above)
Comment (satazor) [reply README.md]: Perhaps it would be useful to add a link to https://github.com/ipfs-shipyard/pm-idm/milestones as well.

File: README.md [hunk 4]
```diff
@@ -1,9 +1,54 @@
...
+
+## Team
+- [André Cruz](https://github.com/satazor) - Software engineer
```
Comment (satazor) [inline README.md]: Add a new line here.

File: README.md [hunk 5]
```diff
@@ -1,9 +1,54 @@
...
+- [André Sousa](https://github.com/andreforsousa) - Designer
+- [Marco Oliveira](https://github.com/marcooliveira) - Product engineer
```
Comment (satazor) [inline README.md]: Remove Marco Oliveira as he is not part of the team, although he will be providing feedback.

Comment (satazor) [inline README.md]: I would add Catia Pereira too.

File: README.md [hunk 6]
```diff
@@ -1,9 +1,54 @@
...
+- [Paulo Marcos](https://github.com/paulobmarcos) - Software engineer
+- [Pedro Santos](https://github.com/PedroMiguelSS) - Software engineer
+- [Pedro Teixeira](https://github.com/pgte) - Adviser
+
+## Milestones & Progress
+
+Our milestones are continually updated and detailed in the [IDM Workplan Document](https://docs.google.com/spreadsheets/d/1Venqgkcao2Lcje0mkCxr9u0H037aC3H5IxMfsoaeMoE).
+
+On the first monday of every sprint we will have a remote call, at 4:00 PM GMT, to report the project's current progress.
+If you care to join us you can do so by following this [link]().
```
Comment (satazor) [inline README.md]: I would remove `If you care to join us you can do so by following this [link]().` and replace with something like: `We create an issue with the label [`progress-call`](https://github.com/ipfs-shipyard/pm-idm/labels/progress-call) for each scheduled call, containing a link to the agenda & notes and with instructions to join to video call`.

File: README.md [hunk 7]
```diff
@@ -1,9 +1,54 @@
...
+## Contributing
+
+All work is organised on GitHub throught this repository. There is also a [Project Board](https://github.com/ipfs-shipyard/pm-idm/projects/1) that we use for organisation, prioritisation and sprint planning.
+
+To manage our project, we adopted the Scrum methodology with two-week sprints. 
+Every sprint begins on a monday with the sprint planning meeting. Additionally, on the second monday of each sprint, there is a backlog grooming session.
+
+The best way to contribute would be to open a [GitHub Issue](https://github.com/ipfs-shipyard/pm-idm/issues) and, if you are willing to, open a [Pull Request](https://github.com/ipfs-shipyard/pm-idm/pulls) while targeting the respective issue. _Also please take in consideration that every commit should be following [Conventional Commits](https://conventionalcommits.org/) guidelines._
+
+You are also welcome to join us in a call at the beginning of every sprint by following this [link]().
```
Comment (satazor) [inline README.md]: Adapt this lane based on my comment above.

File: README.md [hunk 8]
```diff
@@ -1,9 +1,54 @@
...
+All work is organised on GitHub throught this repository. There is also a [Project Board](https://github.com/ipfs-shipyard/pm-idm/projects/1) that we use for organisation, prioritisation and sprint planning.
+
+To manage our project, we adopted the Scrum methodology with two-week sprints. 
+Every sprint begins on a monday with the sprint planning meeting. Additionally, on the second monday of each sprint, there is a backlog grooming session.
+
+The best way to contribute would be to open a [GitHub Issue](https://github.com/ipfs-shipyard/pm-idm/issues) and, if you are willing to, open a [Pull Request](https://github.com/ipfs-shipyard/pm-idm/pulls) while targeting the respective issue. _Also please take in consideration that every commit should be following [Conventional Commits](https://conventionalcommits.org/) guidelines._
+
+You are also welcome to join us in a call at the beginning of every sprint by following this [link](). 
+
+### Codebase
```
Comment (satazor) [inline README.md]: Add new line below.

File: README.md [hunk 9]
```diff
@@ -1,9 +1,54 @@
...
+- [idm-web](https://github.com/idm-web) - IDM Wallet Web GUI.
+- [idm-web-ui-kit](https://github.com/idm-web-ui-kit) - React shared components used by idm-web.
+
+## References
+
+- [DID](https://w3c-ccg.github.io/did-spec/)
+- [IPID](https://github.com/jonnycrunch/ipid)
+- [Verifiable Credentials](https://w3c.github.io/vc-data-model/)
+- [DID Auth](https://github.com/WebOfTrustInfo/rwot6-santabarbara/blob/master/final-documents/did-auth.pdf)
+- [Decentralized Key Management System](https://github.com/hyperledger/indy-sdk/blob/677a0439487a1b7ce64c2e62671ed3e0079cc11f/doc/design/005-dkms/DKMS%20Design%20and%20Architecture%20V3.md)
```
Comment (satazor) [inline README.md]: I would not include this as it's specific to sovrin.

File: README.md (hunk 7 above)
Comment (PedroMiguelSS) [reply README.md]: If this line is related to line number 26, I think it's not necessary.

File: README.md [hunk 10]
```diff
@@ -1,9 +1,54 @@
...
+
+On the first monday of every sprint we will have a remote call, at 4:00 PM GMT, to report the project's current progress.
+If you care to join us you can do so by following this [link]().
+
+## Contributing
+
+All work is organised on GitHub throught this repository. There is also a [Project Board](https://github.com/ipfs-shipyard/pm-idm/projects/1) that we use for organisation, prioritisation and sprint planning.
+
+To manage our project, we adopted the Scrum methodology with two-week sprints. 
+Every sprint begins on a monday with the sprint planning meeting. Additionally, on the second monday of each sprint, there is a backlog grooming session.
```
Comment (satazor) [inline README.md]: I would mention the sprint review and retro as well.

File: README.md (hunk 3 above)
Comment (satazor) [reply README.md]: Also add the Information Architecture document (is an image but it's also a document). See: https://github.com/ipfs-shipyard/pm-idm/issues/7#issuecomment-462456778

**Pull request merged**